packages/kit/src/runtime/server/page/render.js

```diff
@@ -182,6 +182,19 @@ export async function render_response({
 				`Data returned from \`load\` while rendering ${event.routeId} is not serializable: ${error.message} (data.${match[2]})`
 			);
 		}
+
+		const nonPojoError = /pojo/i.exec(error.message);
+
+		if (nonPojoError) {
+			const constructorName = branch[0]?.server_data?.data?.constructor?.name;
```

Comment:
This could be from server_data deeper in the load tree, so I think it should be 
```suggestion
			const constructorName = branch.find(({ server_data }) => server_data?.data?.constructor?.name)?.server_data.data.constructor.name;
```